<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewPeopleWindow</class>
 <widget class="QDialog" name="addNewPeopleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增人物</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 74, 106);</string>
  </property>
  <widget class="QTextEdit" name="addNewPeopleName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="addNewPeopleDescribe">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>561</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="addNewPeopleConnection">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>560</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 181, 205);
color: rgb(255, 255, 255);
border: 0px;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>560</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 181, 205);
color: rgb(255, 255, 255);
border: 0px;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="peoNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="peoDescLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>介绍：</string>
   </property>
  </widget>
  <widget class="QLabel" name="peoConnecLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
